packages/fetchai/skills/tac_control_contract/game.py

```diff
@@ -87,8 +87,9 @@ def __init__(
         self._version_id = version_id
         self._tx_fee = tx_fee
         self._agent_addr_to_name: Dict[str, str] = defaultdict()
```

Comment:
Also I initialise the `agent_addr_to_name` as a `defaultdict()` in order to initialise it from the behaviour without waiting the setup phase. It takes time to deploy the contract and generate the items. So we need to setup the contract before the registration.